config/main.py

```diff
@@ -294,6 +294,22 @@ def _get_device_type():
 
     return device_type
 
+def _get_hwsku(config_db):
```

Comment:
We can remove this function here. Same function available in sonic_py_common  [get_hwsku](https://github.com/Azure/sonic-buildimage/blob/85a631442418582d1a2f77bf0ca1b7b5addee12a/src/sonic-py-common/sonic_py_common/device_info.py#L110)

Reply:
`get_hwsku` in `sonic_py_common` does not support namespaces, it will always connect to the host redis instance.

This can be revisited when `sonic_py_common` provides support for namespaces.

Reply:
hwsku is for the whole device, wouldn't getting the hwsku form the host redis work will it change for each namespace/asic ?

Reply:
This would be the same hwsku for all namespace/asic. However we can't get it before minigraph.xml is parsed and CONFIG_DB is populated, so calling `get_hwsku` from `sonic_py_common` beforehand is not possible since CONFIG_DB will be empty.

Reply:
if config_db is empty then `_get_hwsku`  won't return the correct hwsku  also. I am trying to understand what the difference between `_get_hwsku`  define here and `get_hwsku ` in `sonic_py_common `

Reply:
`_get_hwsku` is called after minigraph.xml is parsed and CONFIG_DB is populated for the corresponding namespace CONFIG_DB initialized at line 1274, _get_hwsku is called at line 1287).

The difference with `get_hwsku` from `sonic_py_common` is that `_get_hwsku` here reuse the existing connection to CONFIG_DB. If we cann `get_hwsku` from `sonic_py_common` instead it will get stuck because we can't open multiple connections the same time.